.github/workflows/ci.yml

```diff
@@ -44,4 +44,5 @@ jobs:
       - name: Script
         run: |
           export PATH=$OPENRESTY_PREFIX/nginx/sbin:$PATH
+          export GODEBUG=x509sha1=1
```

Comment:
Why do this?

Reply:
After updating the package, the certs in the tests cannot be used as the tests fail with an error that its insecure to use sha1 algorithm[1]. This runtime flag skips the check[2]

1. https://github.com/golang/go/issues/41682
2. https://github.com/golang/go/issues/56438

Reply:
Either this, or we would need to regenerate certificates for tests using something like SHA256. Given these are just tests, I think using this flag is fine.